### PR TITLE
Renaming TDE and KeyVaultKey cmdlets from "ManagedInstance" -> "Instance"

### PR DESCRIPTION
**Pull request by petrajkogit**
Renaming TDE and KeyVaultKey cmdlets from "ManagedInstance" -> "Instance"

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@petrajkogit Hey Perica, what is the plan for this PR? Is there a corresponding cmdlet design review for these changes?

**Comment by nivimsft** [comment]
Looks good. However, as per our IM conversation. Some of the tests are disabled because they required pre-existing Managed instance to execute and powershell team did not allow these to be on, so I had to turn them off. Some tests were turned off because they prolonged the continuous validation time.  However, all those tests exercise important use cases and it is crucial to run them before checking in. I would advice running those tests locally before you check your code in.

**Comment by petrajkogit** [comment]
> @petrajkogit Hey Perica, what is the plan for this PR? Is there a corresponding cmdlet design review for these changes?

@cormacpayne There is no design review specifically for this review, since this is just doing renaming of existing cmdlets (to align with MI CRUD cmdlets) and some parameters. I guess there was appropriate design review when initial changes were created, but it was not done by me. I put "do not merge" tag only to make sure @nivimsft takes a look and approves changes before merging, since he wrote them initially. I need to run few more test locally as Nikhil suggested.

File: src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceKeyVaultKey.md [hunk 1]
```diff
@@ -117,25 +117,8 @@ Accept pipeline input: False
...
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
-### -ManagedInstance
-The managed instance input object
-
-```yaml
+### -Instance
+The instance input object```yaml
```
Comment (maddieclayton) [inline src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceKeyVaultKey.md]: Please move yaml tag to a new line.  Apply everywhere. If you upgrade to the latest version of platyPS you should not have this issue.

File: src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceTransparentDataEncryptionCertificate.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ PS C:\> Add-AzureRmSqlManagedInstanceTransparentDataEncryptionCertificate -Resou
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```
Comment (maddieclayton) [inline src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceTransparentDataEncryptionCertificate.md]: Please upgrade to the latest platyPS and run with the -UseFullTypeName parameter: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module

File: src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/GetAzureRmSqlManagedInstanceKeyVaultKey.cs [hunk 3]
```diff
@@ -29,9 +29,9 @@
 namespace Microsoft.Azure.Commands.Sql.TransparentDataEncryption.Cmdlet
 {
     /// <summary>
-    /// Defines the Get-AzureRmSqlManagedInstanceKeyVaultKey cmdlet
+    /// Defines the Get-AzureRmSqlInstanceKeyVaultKey cmdlet
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlManagedInstanceKeyVaultKey", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceKeyVaultKey", SupportsShouldProcess = true, DefaultParameterSetName = DefaultParameterSet)]
     [OutputType(typeof(AzureRmSqlManagedInstanceKeyVaultKeyModel))]
```
Comment (maddieclayton) [inline src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/GetAzureRmSqlManagedInstanceKeyVaultKey.cs]: Did you mean to leave the models with the old syntax?

Comment (petrajkogit) [reply src/ResourceManager/Sql/Commands.Sql/TransparentDataEncryption/Cmdlet/GetAzureRmSqlManagedInstanceKeyVaultKey.cs]: Yes, models were left intentionally with the old name

**Comment by maddieclayton** [comment]
Closing this as I took care of the rename here: https://github.com/Azure/azure-powershell/pull/8143.  Please open a new PR if there's any issues with my update.

**Pull request closed without merging**